benchmarks/dynamo/common.py

```diff
@@ -3687,9 +3693,9 @@ def run(runner, args, original_dir=None):
             log.warning("torch.cuda.is_available() == False, using CPU")
             args.devices = ["cpu"]
 
-    if args.devices != ["cpu"] and torch.cuda.is_available():
+    if args.devices != ["cpu"] and (torch.cuda.is_available() or torch.xpu.is_available()):
```

Comment:
```suggestion
    if args.devices != ["cpu"] and (HAS_CUDA or HAS_XPU):
        device_type = "cuda" if HAS_CUDA else "xpu" if HAS_XPU else ""
```

Reply:
Thanks! The this is changed in https://github.com/pytorch/pytorch/pull/124709/commits/a0b82c81301a0d819a9589f3c0961b91d4c5eeca